extension/js/agent/utils/sendMessage.js

```diff
@@ -29,7 +29,8 @@
     },
 
     sendBatchImmediate: function() {
-      this.send();
+      Agent.postMessage(_.clone(this.queue));
```

Comment:
why clone if you're about to change the reference?

Reply:
I was following the pattern in sendBatch but you are right, is not needed. Changed